src/IconButton/IconButton.js

```diff
@@ -200,9 +223,16 @@ class IconButton extends Component {
     const styles = getStyles(this.props, this.context);
     const tooltipPosition = tooltipPositionProp.split('-');
 
+    const hovered = (this.state.hovered || this.state.isKeyboardFocused) && !disabled;
+
+    const mergedRootStyles = Object.assign(
+      styles.root,
+      {backgroundColor: hovered ? hoverColor : transparent},
```

Comment:
probably yes, i added this to make sure it turns to correct color when mouse hovers out.